### DISCUSSION TITLE
Sync to Office 365

### DISCUSSION THREAD

**Discussion by LukeSkywalker993** [teslamate-org/teslamate#2025] [Ideas]
Hi there,
there is a tool which the journeys, parking times etc ,. can transfer to Outlook or O365 in the calendar?

For me it would be super great if I had the travel time from -> to as a private appointment in Outlook.

**Comment by bastelbert** [comment]
Maybe there's a way with Microsoft Power Automate?! https://flow.microsoft.com/de-de/
